### PR TITLE
Install instructions (#15)

### LINKED ISSUE #15: Include install instructions and some minor improvements
I know that for some this is trivial, but I got asked several times how themes can be installed.

I'm thinking to include:
- "manual" instructions (using `wget`, suitable for desktop)
- `git` instructions (suitable for retropie, so user can update just running a `git pull`)

Additionally I'm thinking to:
- move the credits to a Credits section (still in the `README` file)
- move the CHANGELOG to a `CHANGELOG.txt` file (to remove some pollution)
- and if you @HerbFargus allow, include myself in the Credits section too

### PR DESCRIPTION
**Pull request by raelgc**
Fixes #15

### CODE REVIEW

**Comment by raelgc** [comment]
Waiting your review, @HerbFargus :)

File: README.md [hunk 1]
```diff
@@ -101,6 +64,12 @@ The following platforms are using default `simple` images:
 - xbox360
 
 
+## Credits
+
+Theme `tronkyfran` and variations by Tronkyfran, [Herb Fargus](https://github.com/HerbFargus) and [Rael G.C.](https://github.com/raelgc/).
```
Comment (HerbFargus) [inline README.md]: he has a github as well https://github.com/tronkyfran

File: README.md [hunk 2]
```diff
@@ -22,60 +18,27 @@ This theme is available in 4 variations:
 [3]: https://github.com/HerbFargus/es-theme-tronkyfran/tree/dark
 [4]: https://github.com/HerbFargus/es-theme-tronkyfran/tree/dark-optimized
 
-## Changelog
+## Install `tronkyfran`
+
+1. Close EmulationStation.
+
+2. Create folder `~/.emulationstation/themes` if not exists.
+
```
Comment (HerbFargus) [inline README.md]: Minor grammatical

2. Create folder `~/.emulationstation/themes` if it does not exist.

**Comment by raelgc** [comment]
@HerbFargus Fixed!

**Comment by HerbFargus** [comment]
There are very few people that will download this manually as the main branch at least is integrated with the retropie setup script, so I don't know if that's worth noting as the primary download method, but obviously for the alternates like dark those aren't in the script so those will have to be installed manually.  what are your thoughts

**Comment by raelgc** [comment]
Once this is approved, I'll replicate in each branch but pointing to the respective alternative.

**Comment by HerbFargus** [comment]
sounds good, feel free to merge

**Pull request merged**